src/eventbrite-provider.ts

```diff
@@ -9,22 +11,63 @@ function EventbriteProvider(this: any, options: any) {
 
   let API_KEY = ''
 
+  let eventbriteSDK: Sdk
 
-  // seneca.message('role:entity,cmd:load')
-
+  seneca
+    .message('role:entity,cmd:load,base:eventbrite,name:event', loadEvent)
+    .message('role:entity,cmd:save,base:eventbrite,name:event', saveEvent)
 
 
   seneca.prepare(async function(this: any) {
     let out = await this.post('sys:provider,get:key,provider:eventbrite,key:api')
     if (out.ok) {
       API_KEY = out.value
+      eventbriteSDK = Eventbrite({token: API_KEY})
     }
     else {
       this.fail('api-key-missing')
     }
   })
 
+  async function loadEvent(this: any, msg: any) {
+    const q: any = msg.q
+    const eventID: string = q.id
+
+    const event: any = await eventbriteSDK.request(`/events/${eventID}`)
+
+    if(event.id) {
```

Comment:
As per previous PR, this is probably not needed as we can use the eventbrite id directly. Question: is it a number? in that case we should convert to a string, as that is a more universal is format

Reply:
It is a string. Thanks !